webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSSmartling.java

```diff
@@ -60,15 +66,16 @@
     file in smartling.
     */
     private static final int DEFAULT_BATCH_SIZE = 5004;
-    private static final List<String> EN_LOCALE = ImmutableList.of("en");
+    private static final String LOCALE_EN = "en";
 
     private final SmartlingClient smartlingClient;
     private final AssetPathAndTextUnitNameKeys assetPathAndTextUnitNameKeys;
     private final TextUnitSearcher textUnitSearcher; // TODO - Add a cached version of textunitsearcher
```

Comment:
did you identify perf issue with this? I don't think we use a cache anywhere else for the searcher, and I don't remember it being an issue.

Reply:
Oh yeah, I'll remove this because we won't need it.